assets/html/js/search.js

```diff
@@ -67,7 +67,10 @@ update_search
 
 function worker_function(documenterSearchIndex, documenterBaseURL, filters) {
   importScripts(
-    "https://cdn.jsdelivr.net/npm/minisearch@6.1.0/dist/umd/index.min.js"
+    ...[
+      "https://cdn.jsdelivr.net/npm/minisearch@6.1.0/dist/umd/index.min.js",
+      "https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.21/lodash.min.js",
```

Comment:
Hmm. This now duplicates the dependency.. so let's not include `lodash` here (see also https://github.com/JuliaDocs/Documenter.jl/pull/2415#discussion_r1477029876). The implementation is trivial, so let's just remove the comments, and put the implementation here (with appropriate attribution; lodash is MIT).

https://github.com/lodash/lodash/blob/main/src/escape.ts